contracts/ServiceRegistry.sol

```diff
@@ -3,16 +3,17 @@ pragma solidity ^0.8.9;
 
 import {ITalentLayerID} from "./interfaces/ITalentLayerID.sol";
 import {ITalentLayerPlatformID} from "./interfaces/ITalentLayerPlatformID.sol";
+import {ERC2771RecipientUpgradeable} from "./libs/ERC2771RecipientUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 
 /**
  * @title ServiceRegistry Contract
  * @author TalentLayer Team
  */
-contract ServiceRegistry is Initializable, UUPSUpgradeable, OwnableUpgradeable, AccessControlUpgradeable {
+contract ServiceRegistry is Initializable, ERC2771RecipientUpgradeable, UUPSUpgradeable, AccessControlUpgradeable {
```

Comment:
It seems to me that it only uses the `_msgSender()` function and not `msg.sender` directly, which is fine since the function is overridden inside `ServiceRegistry`